src/libraries/System.Private.CoreLib/src/System/LocalAppContextSwitches.cs

```diff
@@ -47,5 +47,23 @@ public static bool SerializationGuard
             [MethodImpl(MethodImplOptions.AggressiveInlining)]
             get => GetCachedSwitchValue("Switch.System.Runtime.Serialization.SerializationGuard", ref s_serializationGuard);
         }
+
+        private static int s_showILOffset;
+        private static bool GetDefaultShowILOffsetSetting()
+        {
+            if (s_showILOffset < 0) return false;
+            if (s_showILOffset > 0) return true;
+
+            bool isSwitchEnabled = AppContextConfigHelper.GetBooleanConfig("Switch.System.Diagnostics.StackTrace.ShowILOffsets", "DOTNET_ILOffsetToStackTrace");
```

Comment:
@danmoseley suggested that we make this on-by-default for now and then consider whether to change the default back based on feedback later in .NET 6. It looks like this `GetBooleanConfig(string,string)` overload currently sets the default as false.

Reply:
I'll change this on-by-default.